pkg/operator/ceph/cluster/cluster.go

```diff
@@ -236,6 +236,9 @@ func (c *ClusterController) initializeCluster(cluster *cluster, clusterObj *ceph
 		}
 		cephUser = cluster.Info.ExternalCred.Username
 	} else {
+		// Populate ClusterInfo with the last value
+		cluster.mons.ClusterInfo = cluster.Info
```

Comment:
I'm not sure the clusterInfo is loaded yet at this point. I believe it's initialized inside `configureLocalCephCluster()`. I'm thinking we need a call to `LoadClusterInfo()` instead of this.